File: programmers/이진수 더하기.py
def solution(bin1, bin2):
    answer = ''
    num = int(bin1, 2) + int(bin2, 2)
    while num // 2 != 0:
        answer += str(num % 2)
        num = num // 2
    answer += str(num)
    return answer[::-1]


print(solution("10", "11"))
print(solution("1001","1111"))

File: programmers/코딩테스트 입문 with Java/이진수더하기.java
package 코딩테스트입문;

/**
 * https://school.programmers.co.kr/learn/courses/30/lessons/120885
 */
public class 이진수더하기 {
    public static void main(String[] args) {
        System.out.println(solution("10","11"));
        System.out.println(solution("1001","1111"));
    }
    public static String solution(String bin1, String bin2) {
        return  Integer.toBinaryString(Integer.valueOf(bin1,2)+Integer.valueOf(bin2,2));
    }
}
